objects/oAbility/Step_0.gml

```diff
@@ -0,0 +1,9 @@
+/// @description Insert description here
+// You can write your code in this editor
+x = sPlayer.x;
+y = sPlayer.y;
+lifetime -= 1;
+image_alpha = image_alpha * (lifetime/maxLifetime);
+if (lifetime == 0){
```

Comment:
Could use `lifetime <= 0` here just for safety

Reply:
Fixed it